src/components/data/listing/RowDotMenu.js

```diff
@@ -111,6 +116,54 @@ function RowDotMenu(props) {
                         onClose={onClose}
                     />
                 ),
+                !inTrash && [
+                    <CopyLinkMenuItem
+                        key={build(baseId, ids.COPY_LINK_MENU_ITEM)}
+                        baseId={baseId}
+                        onClose={onClose}
+                        onCopyLinkSelected={() => {
+                            const link = `${getHost()}/${
+                                NavigationConstants.DATA
+                            }/${constants.DATA_STORE_STORAGE_ID}${
+                                resource?.path
+                            }?type=${resource?.type}&resourceId=${
+                                resource?.id
+                            }`;
```

Comment:
This is a bit hard to parse. Maybe it could be refactored into a data util function.

In fact, this string building is repeated at least once, in the data search results changes included in this PR.